stacks_queues/stack.py

```diff
@@ -21,13 +21,13 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.length() == 0
```

Comment:
✨ However, you could consider taking advantage of the `empty` method in `LinkedList`